go/libkb/merkle_client.go

```diff
@@ -1159,7 +1241,35 @@ func (vp *VerificationPath) verifyUser(ctx context.Context) (user *MerkleUserLea
 
 	user, err = parseMerkleUserLeaf(ctx, leaf, vp.G())
 	if user != nil {
-		user.uid = vp.uid
+		user.uid = vp.userInfo.uid
+	}
+	return
+}
+
+func (vp *VerificationPath) verifyTeam(ctx context.Context, teamID keybase1.TeamID) (teamLeaf *MerkleTeamLeaf, err error) {
+	curr := vp.root.RootHash()
+
+	var leaf *jsonw.Wrapper
+	leaf, err = vp.path.VerifyPath(curr, teamID.String())
```

Comment:
Ah right, the "this user had never specified an eldest key" condition. To do with that "this user has specified an eldest key but not actually signed anything yet" condition that actually doesn't exist anymore :|